clc/paths/paths.go

```diff
@@ -0,0 +1,87 @@
+package paths
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	unixDir       = ".local/share/clc"
+	EnvCLCHome    = "CLC_HOME"
+	DefaultConfig = "config.yaml"
+)
+
+func Home() string {
+	dir := os.Getenv(EnvCLCHome)
+	if dir == "" {
+		cd, err := os.UserHomeDir()
+		if err != nil {
+			panic(err)
+		}
+		dir = filepath.Join(cd, unixDir)
+	}
+	return dir
+}
+
+func Configs() string {
+	return filepath.Join(Home(), "configs")
+}
+
+func Schemas() string {
+	return filepath.Join(Home(), "schemas")
+}
+
+func Logs() string {
+	return filepath.Join(Home(), "logs")
+}
+
+func DefaultConfigPath() string {
+	return filepath.Join(Configs(), "default", "config.yaml")
+}
```

Comment:
We have only one default config, is it a good practive to create another folder for default?

Reply:
In the new configuration scheme, there one or more configuration directories under `$CLC_HOME/configs`, like:
```
$ tree -d `clc home configs`
/home/yuce/.local/share/clc/configs
├── default
├── pr-1272
├── pr-3066
└── _pr-3066.BAC
```
`default/config.yaml` is the default configuration.